src/GeneExpressionTSnePlot.js

```diff
@@ -123,18 +171,57 @@ const GeneExpressionScatterPlot = (props) => {
                        defaultSpecies={speciesName}
                        onSelect={ (event) => { onSelectGeneId(event) } }
     />,
-
+    //react-responsive design
+    <Desktop key={`Desktop`}>
     <ScatterPlotLoader key={`expression-plot`}
                        wrapperClassName={`row`}
-                       chartClassName={chartClassName}
+                       chartClassName={`small-12 columns`}
                        series={_colourizeExpressionLevel(expressionGradientColours, highlightClusters)(plotData)}
                        highchartsConfig={highchartsConfig}
-                       children={gradient}
                        loading={loading}
+                       legendWidth={1800/2.2}
                        resourcesUrl={resourcesUrl}
                        errorMessage={errorMessage}
     />
-  ]
+    </Desktop>,
+    <Tablet key={`Tablet`}>
+    <ScatterPlotLoader key={`expression-plot`}
+                       wrapperClassName={`row`}
+                       chartClassName={`small-12 columns`}
+                       series={_colourizeExpressionLevel(expressionGradientColours, highlightClusters)(plotData)}
+                       highchartsConfig={highchartsConfig}
+                       loading={loading}
+                       legendWidth={1000/2.2}
```

Comment:
Where do these magic numbers come from? Especial the division by 2.2.

Reply:
This is an empirical value to make sure the legend's width is less than a half of the window's width.